src/github.com/getlantern/flashlight/pac.go

```diff
@@ -41,16 +44,33 @@ func setUpPacTool() error {
 	return nil
 }
 
+func setProxyAddr(addr string) {
```

Comment:
For a given session it will only test once right? I.e., if it decides a site is unblocked, it doesn't keep re-checking right? If so then I don't think putting it in the PAC file would be any different as long as it doesn't persist across sessions.